.github/workflows/ci.yml

```diff
@@ -130,7 +130,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [14, 12, 10, 8, 6]
+        node-version: [14, 13.1, 12, 10, 8, 6]
```

Comment:
We can either keep or remove this, but I first want to see the :heavy_check_mark: from CI

Reply:
Ok I'm removing this since it makes our tests fail for other reasons, but the good news is that the failures are _not_ related to `@babel/runtime` :joy: 

https://github.com/babel/babel/pull/12877/checks?check_run_id=1963171681